CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 1.6.10.2
+* Add support for net5.0 (#85).
+* Use ILRepack instead of ILMerge so .NETFramework builds on all platforms (#92).
+* Many CI and build improvements (#101, #96, #89, #100, #95).
```

Comment:
Worth mentioning that we test multiple versions of Consul after these changes.

Reply:
and maybe worth mentioning that we now actually test with Mono too on Unix platforms?

Reply:
Done!